src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: dynamicMaxWidth($day-width, $gap, $days-in-row, $calendar-padding);
+  padding: $calendar-padding;
+  padding-top: 40px;
+  margin: 0 auto;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    @include cell-dimensions($day-width);
+    border: 1px solid black;
+    transition-duration: 0.5s;
```

Comment:
you should specify properties for which you set transition

Reply:
@silchyn Will do! Thanks!

Reply:
@silchyn Done!